samples/dashboards/edit-mode/ctx-menu-width/demo.css

```diff
@@ -0,0 +1,3 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/css/datagrid.css");
```

Comment:
The same as above, the `datagrid.css` is unnecessary.

Please remove it in each demo, where the dataGrid component is not used.